osu.Game/Rulesets/UI/ModIcon.cs

```diff
@@ -109,18 +118,29 @@ protected override void LoadComplete()
 
         private void updateMod(IMod value)
         {
-            modAcronym.Text = value.Acronym;
-            modIcon.Icon = value.Icon ?? FontAwesome.Solid.Question;
+            var foregroundColour = value.Type == ModType.System ? Color4Extensions.FromHex(@"fc2") : colours.Gray5;
```

Comment:
If yellow is to be the foreground for system mods, then `ModSwitch{Small,Tiny}` may need an update too. There are test scenes for both.